integrated_channels/sap_success_factors/transmitters/courses.py

```diff
@@ -0,0 +1,30 @@
+"""
+Class for transmitting course data to SuccessFactors.
+"""
+from __future__ import unicode_literals
+from integrated_channels.sap_success_factors.transmitters import BaseSuccessFactorsTransmitter
+
+
+class SuccessFactorsCourseTransmitter(BaseSuccessFactorsTransmitter):
+    """
+    This endpoint is intended to carry out an export of course data to SuccessFactors for a given Enterprise.
```

Comment:
It will be course data - the api accepts a list of schedules, so we can send multiple course runs for a single course object on their side.